libs/core/src/lib/select/select-mobile/select-mobile/select-mobile.component.ts

```diff
@@ -52,41 +78,33 @@ export class SelectMobileComponent implements OnInit, AfterViewInit, OnDestroy {
 
     /** @hidden */
     get mobileConfig(): MobileModeConfig {
-        return this._selectComponent.mobileConfig || {};
+        return this._parent.mobileConfig || {};
     }
 
-    /** @hidden */
-    cancel(): void {
-        this._selectComponent.setSelectedOption({
-            option: this._selectedOnOpen,
-            controlChange: !!this._selectedOnOpen
-        });
-        this._selectComponent.close();
+    get isDismissable(): boolean {
+        return !!!this.mobileConfig.approveButtonText;
```

Comment:
Triple negation? 🤔

Reply:
> Should we base `isDismissable` on the fact that approve button text is set?
@JKMarkowski 
The was the previous behavior was based on this, but I will rename it to Ccancalable not to confuse you from original naming

Reply:
Even cancalable sounds wierds. let;s call it hasApproveButton